src/tests/sqlsmith/src/sql_gen/scalar.rs

```diff
@@ -41,9 +41,10 @@ impl<'a, R: Rng> SqlGenerator<'a, R> {
         // e.g. -1 becomes -(1).
         // See: https://github.com/risingwavelabs/risingwave/issues/4344
         match *typ {
-            T::Int64 => Expr::Nested(Box::new(Expr::Value(Value::Number(
-                self.gen_int(i64::MIN as isize, i64::MAX as isize),
-            )))),
+            T::Int64 => Expr::Nested(Box::new(Expr::TypedString {
```

Comment:
Does this mean that the query will look like
`select 10213535::bigint;` instead of `select 10213535`?

Reply:
select bigint '1';